eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -383,6 +383,20 @@ jobs:
 
     # Only run tests if enabled
     - ${{ if eq(parameters.runTests, 'True') }}:
+      # Setup the NuGet sources used by the tests to use private feeds. This is necessary when testing internal-only product
+      # builds where the packages are only available in the private feeds. This allows the tests to restore from those feeds.
+      - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+        - task: Bash@3
+          displayName: Setup Private Feeds Credentials
+          inputs:
+            filePath: $(sourcesPath)/src/sdk/eng/common/SetupNugetSources.sh
```

Comment:
(Orthogonal comment) Isn't this going to be needed for the unified build when we do official release builds (non-source-build)?  It would be needed before the build.

Reply:
Logged https://github.com/dotnet/source-build/issues/4584 for this.